indico/modules/files/controllers.py

```diff
@@ -42,6 +42,8 @@ def _save_file(self, file, stream):
         content_type = mimetypes.guess_type(file.filename)[0] or file.mimetype or 'application/octet-stream'
         f = File(filename=file.filename, content_type=content_type)
         f.save(context, stream)
+        if f.size == 0:
+            raise UnprocessableEntity
```

Comment:
this seems to work fine, we don't need to delete if we raise the exception before adding to the db right @ThiefMaster ?